### PR TITLE
[Improve][Connector-V2] oracle cdc e2e optimization

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
### Purpose of this pull request

oracle cdc e2e optimization

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
I believe the reason is connection leak. Not resource not enough.
image

**Comment by Carl-Zhou-CN** [comment]
@Hisoka-X Yeah, it shouldn't be about resources

File: .github/workflows/backend.yml [hunk 1]
```diff
@@ -1092,4 +1092,26 @@ jobs:
...
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: run doris connector integration test
```
Comment (zhilinli123) [inline .github/workflows/backend.yml]: doris?

Comment (Carl-Zhou-CN) [reply .github/workflows/backend.yml]: done

**Review by hailin0 — COMMENTED** [review]
LGTM

cc @Hisoka-X @EricJoy2048

**Review by hailin0 — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**